### ISSUE TITLE
class="active" doesn't get removed when aria-disabled="true"

### ISSUE THREAD

**Issue by asarro** [onokumus/metismenu#130]
Hi,
I've set the aria-disabled="true" to disable the submenu expanding but the given element was selected and so the class is set to "active". But when I click something else in the menu, it doesn't remove the class="active" and so I end up with two things set as "active".

If you look below "testId-Element" and "testId-Element2" are both active. Is this an expected behaviour?

![capture](https://cloud.githubusercontent.com/assets/5725101/23653906/3a968a60-0326-11e7-9803-62b5e69a769d.PNG)

**Comment by onokumus** [comment]
@asarro 

Yes,  you should add  "active" class to the "li" tag. metisMenu does not make any changes on the "active" class that is added on "a" tag.

**Comment by asarro** [comment]
@onokumus 
hi Osman, thanks for the quick response. I did your suggestion and I set the "li" tag to class="active" but it doesn't seem to change it either if I select another element on the menu.

**Comment by asarro** [comment]
though can I clarify, that when aria-disabled="true" the event "show.metisMenu" doesnt get triggered?
